### PR TITLE
created inventory sources list

### PR DESCRIPTION
**Pull request by resoluteCoder**
Jira ticket: https://issues.redhat.com/browse/AAP-8166

Before:
image

After:
image

### CODE REVIEW

File: frontend/awx/resources/sources/hooks/useSourcesFilters.tsx
Comment (prat98) [inline frontend/awx/resources/sources/hooks/useSourcesFilters.tsx:1]: Not sure if this is being used anywhere

**Review by prat98 — COMMENTED** [review]
PR looks good just a few small changes I could find :)

**Comment by resoluteCoder** [comment]
@jamestalton Updated

**Review by prat98 — APPROVED** [review]
Changes look good to me!

**Pull request merged**